BootloaderCorePkg/Include/HashStore.h

```diff
@@ -22,6 +22,7 @@ typedef struct {
   UINT32             Signature;
   UINT32             Valid;
   HASH_STORE_DATA    Data[HASH_INDEX_MAX_NUM];
+  UINT8              HashType;
```

Comment:
I would recommend to store HashType in HASH_STORE_DATA to be able to accomodate future use-cases.

Reply:
yes, I agree.  Also should we add a revision field for extension in future ?

Reply:
Updated HashStore data structs,,

typedef struct {
  UINT8  Data[HASH_DIGEST_MAX];
  UINT8  KeyUsage;
} HASH_STORE_DATA;

typedef struct {
  UINT32             Signature;
  UINT8 			 Revison;
  UINT8              Valid;
  UINT8              HashAlgoType;
  UINT8              Reserved;
  HASH_STORE_DATA    Data[HASH_INDEX_MAX_NUM];
} HASH_STORE_TABLE;

Reply:
pls sync with previous comments on the structure.

Reply:
This would be structured for variable digest length to optimize the storage.